### PR TITLE
LPS-161917, LPS-161918, LPS-161919 - ObjectNotifications Automated Tests

### PR DESCRIPTION
**Pull request by alanafergs**
https://issues.liferay.com/browse/LPS-161917
https://issues.liferay.com/browse/LPS-161918
https://issues.liferay.com/browse/LPS-161919

### CODE REVIEW

**Comment by alanafergs** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectNotifications.path [hunk 1]
```diff
@@ -20,6 +20,17 @@
...
+	<td>DROPDOW_MENU</td>
+	<td>//div[@class='table-list-title' and contains(., '${key_templateName}')]//..//..//button[@class='dropdown-toggle component-action dropdown-toggle ml-1 btn btn-unstyled']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DROPDOW_MENU_ITEM</td>
+	<td>//div[@class='dropdown-menu show']//button[@class='dropdown-item' and contains(., '${key_menuItem}')]</td>
+	<td></td>
+</tr>
+
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectNotifications.path:33]: We should have an appropriate path for the ellipsis in `Icon.path` and there's `MenuItem.path` for menu items.

**Review by timpak — CHANGES_REQUESTED** [review]
Congrats on your first Objects tests! Please see my comments for changes needed. Also, for SF changes, please make them in the commit that caused them instead of making a separate commit. In general, it's good practice to fix SF errors in the commit that caused them instead of fixing them in a separate commit.

Let me know if you need help with this!

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase [hunk 2]
```diff
@@ -83,4 +83,36 @@ definition {
 		}
 	}
 
+     @description = "LPS-161918 - Verify that it's possible to edit an e-mail notification template"
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase]: Typo, should be `delete` instead of `edit`

**Pull request closed without merging**